src/test/java/CalculatorTest.java

```diff
@@ -4,28 +4,42 @@
 
 class CalculatorTest {
 
+    Calculator calculator = new Calculator();
+
     @Test
     void add() {
-        //TODO inser your realisation in test-method add
+        assertEquals(10, calculator.add(7, 3));
+        assertEquals(0, calculator.add(0, 0));
+        assertEquals(2, calculator.add(1, 1));
     }
 
     @Test
     void dif() {
-        //TODO inser your realisation in test-method dif
+        assertEquals(5, calculator.dif(8, 3));
+        assertEquals(4, calculator.dif(10, 6));
+        assertEquals(-3, calculator.dif(5, 8));
     }
 
     @Test
     void div() {
-        //TODO inser your realisation in test-method div
+        assertEquals(4, calculator.div(24, 6));
+        assertEquals(2, calculator.div(10, 5));
+        assertThrows(IllegalArgumentException.class,
+                () -> calculator.div(10, 0), "You can't divide by 0");
     }
 
     @Test
     void times() {
-        //TODO inser your realisation in test-method times
+        assertEquals(9, calculator.times(9, 3));
+        assertEquals(-1, calculator.times(-1, 1));
+        assertEquals(0, calculator.times(0, 10));
     }
 
     @Test
     void solver() {
-        //TODO inser your realisation in test-method solver
+        assertEquals(18, calculator.solver(4, 5, 8, 2));
+        assertEquals(11, calculator.solver(3, 4, 4, 2));
+        assertThrows(IllegalArgumentException.class,
+                () -> calculator.solver(3, 4, 4, 0), "You can't divide by 0");
     }
```

Comment:
These methods are not tested

Reply:
Oops...

Reply:
![image](https://github.com/user-attachments/assets/4b3123c4-fe08-4474-a885-e63543d26a29)